src/RecoveryServices/RecoveryServices.Backup.Autorest/test/Edit-AzrecoveryServicesBackupPolicyClientObject.Tests.ps1

```diff
@@ -0,0 +1,37 @@
+if(($null -eq $TestName) -or ($TestName -contains 'Edit-AzrecoveryServicesBackupPolicyClientObject'))
+{
+  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+  if (-Not (Test-Path -Path $loadEnvPath)) {
+      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+  }
+  . ($loadEnvPath)
+  $TestRecordingFile = Join-Path $PSScriptRoot 'Edit-AzrecoveryServicesBackupPolicyClientObject.Recording.json'
+  $currentPath = $PSScriptRoot
+  while(-not $mockingPath) {
+      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+      $currentPath = Split-Path -Path $currentPath -Parent
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'Edit-AzrecoveryServicesBackupPolicyClientObject' {
+    It 'IsDailyScheduleEnabled' -skip {
```

Comment:
That's different file I think this would be deleted as new commands are Edit-AzrecoveryServicesBackupRetentionPolicyClientObject and Edit-AzrecoveryServicesBackupSchedulePolicyClientObject

Reply:
yes we can delete this file